sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -641,21 +649,28 @@ else if (mUnknownSourcesDialog != null) {
                 return;
             }
 
-            /* Check if we have previous stored the update token. */
+            /*
+             * Check if we have previously stored the redirection parameters,
+             * note that distribution group is missing in previous SDK releases.
```

Comment:
I don't think we need the 2nd line unless the version is specified. It is saying previous SDK but we don't actually know what the "previous" is.